micrometer-core/src/main/java/io/micrometer/core/instrument/placeholder/Placeholders.java

```diff
@@ -0,0 +1,58 @@
+package io.micrometer.core.instrument.placeholder;
+
+import io.micrometer.core.annotation.Incubating;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.dropwizard.DropwizardMeterRegistry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * An entry-point for the placeholders feature.
+ *
+ * Placeholders are an alternative way to build metric names from tags in hierarchical
+ * registries like Dropwizard (Graphite). Users may put placeholders in the metric name
+ * which define positions in which given tags should be encoded. When bound to a registry
+ * it replaces the default strategy of tag encoding by appending all tag keys and values to the
+ * metric name.
+ *
+ * By defining metric name mappings, users can define placeholders in metrics that are
+ * outside of their control, e.g. standard memory metrics.
+ *
+ * @author Piotr Betkier
+ */
+@Incubating(since = "1.1.0")
```

Comment:
Since it was not included into `1.1.0` probably worth to update when it's clear which version will include these changes